src/lib/featureChecking.ts

```diff
@@ -1,7 +1,7 @@
 import get from 'lodash/get';
 
 type FeatureOptions = {
-    [key: string]: {};
+    [key: string]: NonNullable<unknown>;
```

Comment:
Would `unknown` work here? It seems like this value could easily be `null`.

Reply:
Hmm, it looks like when using `unknown`, the linter pops this error message up  on line 16 of this file:
```
Type 'FeatureOptions | {}' is not assignable to type 'FeatureConfig'.
  Type 'FeatureOptions' is not assignable to type 'FeatureConfig'.
    'string' index signatures are incompatible.
      Type 'unknown' is not assignable to type 'FeatureOptions'.
```